README.md

```diff
@@ -75,6 +75,9 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [Redirect](https://github.com/nuxt-community/redirect-module) - Simplify setting up correct URL redirection with NuxtJS. No more cumbersome redirects!
 - [I18n](https://github.com/nuxt-community/nuxt-i18n) - Add i18n capabilities to your Nuxt project.
 - [Separate Env](https://github.com/nuxt-community/separate-env-module) - Tear apart your variables and split them up for client/server bundle!
+- [nuxt-logrocket](https://github.com/nuxt-community/nuxt-logrocket) - LogRocket module for Nuxt.js
```

Comment:
Isn't it better using friendly titles like `LogRocket` / `Google Optimize` ?

Reply:
@pi0 I copied the names from their GitHub repos, but yeah, I agree.

I changed the names in the most recent commit.